stakingcontract/sc_utils.go

```diff
@@ -0,0 +1,51 @@
+package sc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/Evrynetlabs/evrynet-node"
+	"github.com/Evrynetlabs/evrynet-node/common"
+	"github.com/Evrynetlabs/evrynet-node/core/types"
+	"github.com/Evrynetlabs/evrynet-node/evrclient"
+)
+
+// CheckTransStatus returns stratus of a transaction
```

Comment:
Please add the comment that this function is blocking. Alternatively, we can try make it concurrently by return the result into a channel where the function caller can choose to block it or to parallel it.

Reply:
i added the comment to this function as your request